Rakefile

```diff
@@ -4,11 +4,7 @@ require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
 
-RuboCop::RakeTask.new do |task|
-  task.requires << 'rubocop-rspec'
```

Comment:
do we not need to do this any longer? did it get folded into regular rubocop behavior, or something?

Reply:
yep